pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -306,9 +301,9 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 					TCPCloseWaitTimeout:   &metav1.Duration{Duration: 5 * time.Second},
 				},
 			},
-			msg: "only one CIDR allowed (e.g. 10.100.0.0/16 or fde4:8dba:82e1::/48)",
+			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("ClusterCIDR"), "192.168.59.0/24,fd00:192:168::/64", "only one CIDR allowed (e.g. 10.100.0.0/16 or fde4:8dba:82e1::/48)")},
 		},
-		{
+		"Two IPv4 ClusterCIDRs provided with DualStack feature-enabled": {
```

Comment:
The title doesn't match the test, should it be `DualStack feature-enabled but EndpointSlice feature disabled`?

Reply:
Ah yes, this is something I forgot to update when I rebased